### PR TITLE
zh-CN: Trans for 'editOnGitHub'

### PR DESCRIPTION
**Pull request by ghost**
Just like https://github.com/nodejs/nodejs.org/commit/631e5da553d50392d6344637fe57171936fb1442#diff-c73a453a788f6d072e426628a94d129f. 
We should add this for Chinese version in both Tranditional and
Simplified one.

PS：@nodejs/localization, you can also do translations if you are free :)

### CODE REVIEW

File: locale/zh-cn/site.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "title": "Node.js",
+  "editOnGithub":"在 GitHub 中直接编辑",
```
Comment (ZYSzys) [inline locale/zh-cn/site.json]:
```suggestion
  "editOnGithub":"在 GitHub 上编辑",
```

I would prefer to translate literally.

File: locale/zh-tw/site.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
   "title": "Node.js",
+  "editOnGithub":"在 GitHub 中直接編輯",
```
Comment (ZYSzys) [inline locale/zh-tw/site.json]:
```suggestion
  "editOnGithub":"在 GitHub 上編輯",
```

Ditto

**Review by ZYSzys — APPROVED** [review]
👍

**Review by willin — APPROVED** [review]
LGTM

**Pull request merged**